docs/product-manuals/tasklist/userguide/camunda-forms.md

```diff
@@ -0,0 +1,65 @@
+---
+id: camunda-forms
+title: Camunda Forms
+---
+
+## What are Camunda Forms?
+
+Camunda Forms is one of the handling a User Task on Tasklist. These Forms can be created on the Camunda Modeler where they can also be embedded to the BPMN file with your Diagram and assign a Form to a User Task.
+
+After deploying a Diagram with an embedded Form, Tasklist will import this Form schema to the API where every Task that is assigned to this Form will be related to the Form schema on the API.
```

Comment:
The API I'm referring to is the Tasklist GraphQL API (kind of like a headless Tasklist), but I guess it's not relevant to refer to it on this case since the API is not released yet and this tutorial is more focused on our own Tasklist app.
What do you think about replacing this sentence with:
```
After deploying a diagram with an embedded form, Tasklist will import this form schema
and use it to render the form on every task assigned to it.
```

Reply:
Works for me.